featurebyte/routes/feature/controller.py

```diff
@@ -96,9 +96,13 @@ async def submit_batch_feature_create_task(self, data: BatchFeatureCreate) -> Op
         Optional[Task]
             Task object
         """
+        # as there is no direct way to get the conflict resolved feature id for batch feature creation task,
+        # the conflict resolution should only support "raise" for public API. Therefore, we should not include
+        # the conflict resolution in the API payload schema (BatchFeatureCreate).
```

Comment:
Does this affect whether we can support "retrieve" resolution in any of the SDK feature / feature list creation methods?

Reply:
Technically, both feature & feature list async task can support `retrieve` resolution. The problem is that there is no direct way for feature save to retrieve resolved id. That's why `conflict_resolution` parameter is excluded from batch feature creation POST route. For feature list with batch feature creation task, we can use temporary feature list to obtain resolved feature id. That's why `conflict_resolution` parameter is kept for feature list with batch feature creation POST route.